.github/workflows/pylint.yml

```diff
@@ -1,25 +1,39 @@
-name: Back-End - Pylint
+name: PyLint
 
-on: [ push ]
+on: [pull_request]
 
 jobs:
   build:
+
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ["3.8", "3.9", "3.10"]
+
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
       with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+        python-version: 3.8
+    - name: Install Deps
       run: |
-        python -m pip install --upgrade pip
+        python3 -m pip install --upgrade pip
         pip install pylint
-        pip install wheel
+        pip install pylint-exit
         pip install -r requirements.txt
-    - name: Analysing the code with pylint
+    - name: Analyse Code
       run: |
-        pylint $(git ls-files '*.py')
+        python3 -m pylint $(git ls-files '*.py') > output.txt || echo $? > code.txt
+        cat output.txt
+        cat output.txt | grep -i rated > result.txt
```

Comment:
Could you change the grep statement to grep for:

"Your code has been rated"

This is likely the issue why the comments for the rating look messed up